README.md

```diff
@@ -362,6 +363,38 @@ Alternatively, you can also update the service `awx-service` in your namespace t
 #> minikube service <serviceName> -n <namespaceName> --url
 ```
 
+### Generating a bundle
+
+If one has the Operator Lifecycle Manager (OLM) installed, the following steps is the process to generate the bundle that would nicely display in the OLM interface.
+
+At the root of this directory:
+
+1. Build and publish the operator: `operator-sdk build <registry>/<namespace>/<operator-container>:<tag> && podman push <registry>/<namespace>/<operator-container>:<tag>`
+
+2. Build and publish the bundle: `podman build -f bundle.Dockerfile -t <registry>/<namespace>/<bundle-container>:<tag> . && podman push <registry>/<namespace>/<bundle-container>:<tag>`
+
+3. Build and publish an index with your bundle in it: `opm index add --bundles <registry>/<namespace>/<bundle-container>:<tag>  --tag <registry>/<namespace>/<index-container>:<tag>  && podman push <registry>/<namespace>/<index-container>:<tag> `
```

Comment:
Same here:
```sh
opm index add --bundles registry.example.com/ansible/awx-operator-bundle:mytag  --tag registry.example.com/ansible/awx-operator-catalog:mytag  && podman push registry.example.com/ansible/awx-operator-catalog:mytag
```

Reply:
For the sake of readability I would do something like:

```
$ opm index add --bundles <registry>/<namespace>/<bundle-container>:<tag> --tag <registry>/<namespace>/<index-container>:<tag> 
$ podman push <registry>/<namespace>/<index-container>:<tag> 
```